packages/coverage-c8/src/provider.ts

```diff
@@ -80,11 +80,6 @@ export class C8CoverageProvider implements CoverageProvider {
         }
       }))
 
-    // This is a magic number. It corresponds to the amount of code
-    // that we add in packages/vite-node/src/client.ts:114 (vm.runInThisContext)
-    // TODO: Include our transformations in sourcemaps
-    const offset = 224
```

Comment:
I used `columnOffset` in `vm.runInThisContext`, but I am actually not sure if this works 😄 Would be great, if you can confirm? @Demivan 

If it doesn't work, I can revert the change

Reply:
I have checked it with my project. It doesn't work. Entire else (74-95) should be not covered:

<details>
  <summary>Screenshot</summary>
  
![image](https://user-images.githubusercontent.com/2339406/199193540-74751f4d-9391-4ae9-9d39-bc519a811967.png)
</details>

Reply:
reverted the change

we should probably create a test for this 😄